src/components/Dialog/ImportSongProjectDialog.vue

```diff
@@ -0,0 +1,268 @@
+          <QToolbarTitle class="text-display"
+            >プロジェクトファイルのインポート</QToolbarTitle
+          >
+        </QToolbar>
+      </QHeader>
+      <QPageContainer class="q-px-lg">
+        <p class="q-pt-md">以下のプロジェクトファイルをインポートできます：</p>
+        <ul>
+          <li v-for="[name, extensions] in projectNameToExtensions" :key="name">
+            {{ name }}：{{ extensions.map((ext) => `.${ext}`).join("、") }}
+          </li>
+        </ul>
+        <QFile
+          v-model="projectFile"
+          label="インポートするファイル"
+          class="q-my-sm"
+          :accept="acceptExtensions"
+          :error-message="projectFileError"
+          :error="!!projectFileError"
+          placeholder="外部プロジェクトファイルを選択してください"
+          @input="handleFileChange"
+        />
+        <QSelect
+          v-if="project"
+          v-model="selectedTrack"
+          :options="tracks"
+          :disable="projectFileError != undefined"
+          emit-value
+          map-options
+          label="インポートするトラック"
+        />
+      </QPageContainer>
+      <QFooter>
+        <QToolbar>
+          <QSpace />
+          <QBtn
+            unelevated
+            align="right"
+            label="キャンセル"
+            color="toolbar-button"
+            text-color="toolbar-button-display"
+            class="text-no-wrap text-bold q-mr-sm"
+            @click="handleCancel"
+          />
+          <QBtn
+            unelevated
+            align="right"
+            label="インポート"
+            color="toolbar-button"
+            text-color="toolbar-button-display"
+            class="text-no-wrap text-bold q-mr-sm"
+            :disabled="selectedTrack === null || projectFileError != undefined"
+            @click="handleImportTrack"
+          />
+        </QToolbar>
+      </QFooter>
+    </QLayout>
+  </QDialog>
+</template>
+
+<script setup lang="ts">
+import { computed, ref } from "vue";
+import { useDialogPluginComponent } from "quasar";
+import {
+  Project,
+  EmptyProjectException,
+  IllegalFileException,
+  NotesOverlappingException,
+  supportedExtensions,
+  SupportedExtensions as UfSupportedExtensions,
+} from "@sevenc-nanashi/utaformatix-ts";
+import semver from "semver";
+import { useStore } from "@/store";
+import { createLogger } from "@/domain/frontend/log";
+import { readTextFile } from "@/helpers/fileReader";
+import { migrateProjectFileObject } from "@/domain/project";
+import { ExhaustiveError } from "@/type/utility";
+import { songProjectToUfData } from "@/sing/songProjectToUfData";
+import { IsEqual } from "@/type/utility";
+
+const { dialogRef, onDialogOK, onDialogCancel } = useDialogPluginComponent();
+
+const store = useStore();
+const log = createLogger("ImportExternalProjectDialog");
+
+// 受け入れる拡張子
+const acceptExtensions = computed(
+  () => supportedExtensions.map((ext) => `.${ext}`).join(",") + ",.vvproj",
+);
+
+type SupportedExtensions = UfSupportedExtensions | "vvproj";
+
+const projectNameToExtensions = [
+  ["Cevio AI", ["ccs"]],
```

Comment:
表記はこうかな。　あとAIじゃないのもある気がするのでAI消しても良いかも。あっても良いけど。
```suggestion
  ["CeVIO", ["ccs"]],
```